ubuntu_update.go

```diff
@@ -0,0 +1,67 @@
+// USAGE: go run ubuntu_update.go
+// This script generates ubuntu's trust store in a file ubuntu.pem
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+// Change updateURL and packageName as necessary to ubuntu's ca-certificates update
+var updateURL = "http://archive.ubuntu.com/ubuntu/pool/main/c/ca-certificates/"
+var packageName = "ca-certificates_20150426ubuntu1.tar.xz"
+
+// "tar xvf packageName" will create a directory without the ".tar.xz" extension
+// and will also replace all underscores "_" with dashes "-"
+var dirName string = strings.Replace(strings.TrimSuffix(packageName, ".tar.xz"), "_", "-", -1)
+
+func main() {
+	// Download the package to local directory
+	output, _ := os.Create(packageName)
+	defer output.Close()
+
+	res, _ := http.Get(updateURL + packageName)
```

Comment:
@lziest Much appreciated. You find the latest source code of the package here: http://bazaar.launchpad.net/~ubuntu-branches/ubuntu/wily/ca-certificates/wily/files/head:/